jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/telemetry/CodeWhispererCodeCoverageTracker.kt

```diff
@@ -208,7 +229,15 @@ abstract class CodeWhispererCodeCoverageTracker(
 class DefaultCodeWhispererCodeCoverageTracker(language: CodewhispererLanguage) : CodeWhispererCodeCoverageTracker(
     5 * TOTAL_SECONDS_IN_MINUTE,
     language,
-    acceptedTokens = AtomicInteger(0),
-    totalTokens = AtomicInteger(0),
-    mutableListOf()
+    mutableListOf(),
+    mutableMapOf()
 )
+
+class CodeCoverageTokens(totalTokens: Int = 0, acceptedTokens: Int = 0) {
+    val totalTokens: AtomicInteger
+    val acceptedTokens: AtomicInteger
+    init {
+        this.totalTokens = AtomicInteger(totalTokens)
+        this.acceptedTokens = AtomicInteger(acceptedTokens)
```

Comment:
Wonder if we really need this atomicInteger -- Since right now it looks like the usage of these two counts is all from documentChanged() events which happens in Edt. Doesn't hurt to have them though.

Reply:
yes right, this needs to be an AtomicInteger